src/TTimer.cpp

```diff
@@ -371,3 +371,8 @@ void TTimer::killTimer()
     deactivate();
     mpQTimer->stop();
 }
+
+int TTimer::remainingTime()
+{
+    return mpQTimer->remainingTime();
+}
```

Comment:
Missing EOL on the last line of the file (go to the end and put a line-feed on the end - by pressing `<Enter>` in your editor perhaps?) :wink: